toolchain/lc-compile/src/emit.cpp

```diff
@@ -425,7 +425,44 @@ void EmitForeignHandlerDefinition(long p_index, PositionRef p_position, NameRef
     else
         t_binding_str = to_mcstringref(p_binding);
     
-    MCScriptAddForeignHandlerToModule(s_builder, to_mcnameref(p_name), p_type_index, *t_binding_str, p_index);
+    if (strncmp((const char *)p_binding, "lcb:", 4) != 0)
```

Comment:
I know that it's a hassle to convert things to/from `MCStringRef`, and that this code is temporary.  However, I can't help but think that there are some potential bugs that could be avoided by doing string manipulation via the `MCString` API rather than by banging string pointers around.